pkg/fanal/artifact/image/image.go

```diff
@@ -550,3 +527,31 @@ func (a Artifact) guessBaseLayers(diffIDs []string, configFile *v1.ConfigFile) [
 	}
 	return baseDiffIDs
 }
+
+func GuessBaseImageIndex(histories []v1.History) int {
```

Comment:
Can we move it under `fanal/image/`? The existing file `image.go` or a new file, `history.go`

Reply:
I don't think that we need to create new file for 1 function.
I moved function to `fanal/image/image.go` in b2cce5f8feec1067de963f67de57810891d85882